src/obr/signac_wrapper/operations.py

```diff
@@ -260,9 +260,48 @@ def dispatch_pre_hooks(operation_name, job):
     execute_pre_build(operation_name, job)
 
 
+def gather_caseFiles(case_path: str) -> list[str]:
+    """Gathers a list of files inside a case folder that are also not symlinks."""
+    sys_path = case_path + '/system'
+    sys_include_path = sys_path+'/include'
+    const_path = case_path + '/constant'
+    const_polyMesh_path = const_path + '/polyMesh'
+    files = []
+    if os.path.isdir(sys_path):
+        for f in os.listdir(sys_path):
+            f_path = os.path.join(sys_path, f)
+            if os.path.isfile(f_path) and not os.path.islink(f_path):
+                files.append(f_path)
+    if os.path.isdir(sys_include_path):
+        for f in os.listdir(sys_include_path):
+            f_path = os.path.join(sys_include_path, f)
+            if os.path.isfile(f_path) and not os.path.islink(f_path):
+                files.append(f_path)
+    if os.path.isdir(const_path):
+        for f in os.listdir(const_path):
+            f_path = os.path.join(const_path, f)
+            if os.path.isfile(f_path) and not os.path.islink(f_path):
+                files.append(f_path)
+    if os.path.isdir(const_polyMesh_path):
+        for f in os.listdir(const_polyMesh_path):    # NOTE is polymesh subject to change during simulation?
+            f_path = os.path.join(const_polyMesh_path, f)
+            if os.path.isfile(f_path) and not os.path.islink(f_path):
+                files.append(f_path)
+    return files
+
+
 def dispatch_post_hooks(operation_name, job):
     """just forwards to start_job_state and execute_pre_build"""
     execute_post_build(operation_name, job)
+    files = gather_caseFiles(f'{job.path}/case')
+    for case_file in files:
+        md5sum = check_output(["md5sum", str(case_file)], text=True)
+        if 'md5sum' not in job.doc['obr']:
+            job.doc['obr']['md5sum'] = dict()
+        _, r_path = case_file.split(job.id)[:]  # NOTE is total path really necessary?
```

Comment:
If we store it somewhere a relative path (to the project root) would be better so that we can relocate the project.

Reply:
just to be clear, if I had a path like this: 
`micro_benchmarks/WindsorBody/workspace/78e2de3e6205144311d04282001fe21f/case/system/controlDict` , 
you would expect this relative path to be 
`WindsorBody/workspace/78e2de3e6205144311d04282001fe21f/case/system/controlDict` , assuming `WindsorBody` is the directory with the `signac.rc` included?

Reply:
I guess it would be even sufficient to store the path relative to the case root (ie case) it self.